code/modules/clothing/suits/modular_armor.dm

```diff
@@ -878,5 +879,35 @@
 	icon_state = "tcaf_senior_legion_pauldron"
 	item_state = "tcaf_senior_legion_pauldron"
 	contained_sprite = TRUE
-	slot = ACCESSORY_SLOT_GENERIC
+	accessory_slots = ACCESSORY_SLOT_ARMOR
 	flippable = FALSE
+
+// Helmet Accessories
+/obj/item/clothing/accessory/sec_mask
+	name = "corporate protective mask"
+	desc = "A generic camera-assisted mask issued to members of SCC security teams. Mainly used to protect the user \
+	from dust and other irritants, but lacks the capability for much else. While some may complain about the generic \
+	appearance, the Conglomerate continue with their tagline: \"Through uniformity, obedience.\""
+	icon = 'icons/obj/item/clothing/accessory/security_mask.dmi'
+	icon_state = "sec_mask"
+	item_state = "sec_mask"
+	contained_sprite = TRUE
+	accessory_slots = ACCESSORY_SLOT_HELMET
+	/**
+	 * What the mask resembles, displayed after the examine text
+	 */
+	var/design = "This one vaguely resembles a Human face."
```

Comment:
This should be a generic item first, with a SCC/Horizon flavored subtype, so it can be used for offships or events or whatever.

Reply:
It should also hide the wearer's face, seeing as it kinda covers it whole.

Reply:
Not resolved, this is still in the loadout.

Reply:
Should be gone now, I thought I deleted it.

Reply:
Done